src/ensembl/src/content/app/browser/Browser.tsx

```diff
@@ -82,13 +85,16 @@ export const Browser = () => {
   };
 
   const shouldShowNavBar =
-    browserActivated && browserNavOpenState && !isDrawerOpened;
-
-  const mainContent = (
-    <>
-      {shouldShowNavBar && <BrowserNavBar />}
-      <BrowserImage />
-    </>
+    genomeBrowser && browserNavOpenState && !isDrawerOpened;
+
+  const mainContent = useMemo(
+    () => (
+      <>
+        {shouldShowNavBar && <BrowserNavBar />}
+        <BrowserImage />
+      </>
+    ),
+    [shouldShowNavBar]
   );
```

Comment:
Just wondering — wouldn't it be more idiomatic to just export `BrowserImage` and, possibly, `BrowserNavBar` are `memo`-ed components?